pages/docs/manual/latest/pattern-matching-destructuring.mdx

```diff
@@ -580,6 +580,35 @@ Here are some advices.
 
 Do not abuse the wildcard `_` too much. This prevents the compiler from giving you better exhaustiveness check, which would be especially important after a refactoring where you add a new case to a variant. Try only using `_` against infinite possibilities, e.g. string, int, etc.
 
+If you're about to use a wildcard when matching on a large but finite set of possibilities because you have many possibilities that return the same thing, consider reaching for disjunctive patterns instead. That is, you can replace:
+
+```res example
+type t = A | B | C | D | E
+let v = C
+switch v {
+| A => 1
+| B => 2
+| C => 2
+| D => 2
+| E => 2
+}
+```
+
+with:
+
+```res example
+type t = A | B | C | D | E
+let v = C
+switch v {
+| A => 1
+| B
+| C
+| D
+| E => 2
+}
+```
+This still requires a bit more work than a wildcard, but we promise it'll be worth it.
```

Comment:
`we promise it'll be worth it.` -> this is super vague. Can you illustrate/explain this?

Reply:
Also the rescript formatter allows the following kind of formatting:
```
switch v {
| A => 1
| B | C | D | E => 2
}
```